app/controllers/blog_posts_controller.rb

```diff
@@ -24,4 +25,12 @@ def show
   def pro_blog_posts
     Pro::BlogPost.includes(:network, :persons)
   end
+
+  def themed_blog_posts
```

Comment:
There's something peculiar about needing a method like this. Maybe due to the fact that we have to create the pro_blog_posts method in order to create the scope in the first place.

I wonder if it makes sense to move the assignment of the blog_posts for the index action to a before_action like: 
```
  before_action :set_blog_posts, only: :index
```
and then move the logic for determining which posts to retrieve into the set_blog_posts like:
```
  def set_blog_posts
    if blog_posts_theme == 'all'
      @blog_posts = pro_blog_posts.page(pagination_page).per(pagination_per).all
    else
      @blog_posts = pro_blog_posts.where(tags: blog_posts_theme).page(pagination_page).per(pagination_per).all
    end
  end
```
I'm not 100% sure it's stylistically the better option, but it means we wouldn't call themed_blog_posts in a situation where we're not actually retrieving themed blog posts, rather than all blog posts. @rwd What do you think?